docs/Developers/Writing_Good_Commits.rst

```diff
@@ -26,6 +26,9 @@ Example of a good commit:
 - `Fixes https://github.com/coala/coala/issues/5861`: Mention the URL
    of the issue it closes or fixes.
 
+Use 'Fixes' when you fixed a bug and 'Closes' when your work is
```

Comment:
@sahilarora535 Sounds good! But I also like the original sentence and think it would be nice to keep the heritage of `AnaDenisa` alive... whoever she is ;)

@sils Opinion?